WISE-Community/WISE
# Revived HibernateProjectDaoTest

How to test:

1. Run ```mvn test```. 191 test should run and pass.
1. Open HibernateProjectDaoTest.java in VSCode. There should be a "Run | Debug" link above the class definition and test methods if you have installed Java Debugger plugin. Click on them. The test(s) should run and pass.

Note 1: I chose to rename the method names of the unit tests, and also made changes to the unit tests in HibernateRunDaoTest.java. I'm following the suggestions in this blog: https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html

Note 2: I also removed 'this.' in front of variable names because I think they are redundant.
